Numbers concept (#1054)
exercism/haskell: exercises/concept/temperature/.docs/instructions.md

# Instructions

## What's the Temperature?

Write a function `tempToC` to convert an integer temperature in Fahrenheit to its equivalent in Celsius rounded to the nearest hundredth. It should work like this:

```haskell
> tempToC 32
0.00
```

Write a second function, `tempToF` to convert a decimal temperature in Celsius to its equivalent in Fahrenheit. For the safety of your American team mate, round the result to the **next** integer. It should work like this:

```Haskell
> tempToF 0.00
32

> ```

Reviewer: The format used here is slightly different from what we require.
The key thing is that there are no individual tasks being defined (they are grouped together). 
See https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-instructions-md for more information on how to structure the `instructions.md` document.

# Instructions

## What's the Temperature?

You are part of a firefighter organization made of international volunteers. Over the last few calls your team has wasted valuable time translating from the new American team member charged with estimating the temperature inside the burning building. To save time, and people, you decide to create a program to automatically convert to and from Fahrenheit and Celsius for your new teammate.

## 1. Write a function `tempToC` to convert an integer temperature in Fahrenheit to its equivalent in Celsius rounded to the nearest hundredth. It should work like this:

```haskell
> tempToC 32

> 0.00

Author: Changed to 4C ->  39.2F ~ 40C (answer)

# Instructions

> You are part of a firefighter organization made of international volunteers. Over the last few calls your team has wasted valuable time translating from the new American team member charged with estimating the temperature inside the burning building. To save time, and people, you decide to create a program to automatically convert to and from Fahrenheit and Celsius for your new teammate.

Reviewer: We have a "one sentence per line" style guide (see https://exercism.org/docs/building/markdown/markdown#h-one-sentence-per-line):

I've also suggested a slightly rephrased version, to not pin the blame on a specific group (Americans) :)

```suggestion
You are part of a firefighter organization made of international volunteers.
Due to the international nature of the team, valuable time has been wasted over the last few calls translating between Fahrenheit and Celcius while estimating the temperature inside the burning buildings.
To save time, and people, you decide to create a program to automatically convert between Fahrenheit and Celsius for your new teammate.
```

The main benefit of this, is that diffs become much easier to parse.

## 1. Convert Farenheit to Celsius

Write a function `tempToC` to convert an `Integer` temperature in Fahrenheit to its equivalent `Float` in Celsius rounded to the nearest hundredth. 

> Temperature in degrees Celsius is defined as: T(°C) = (T(°F) - 32) / 1.8. Your function should work like this:

Reviewer: I think it's fine to leave off the last bit.

```suggestion
Temperature in degrees Celsius is defined as: T(°C) = (T(°F) - 32) / 1.8.
```

```haskell
> tempToC 32
0.00
```

## 2. Convert Celsius to Farenheit

> Write a second function, `tempToF` to convert a `Float` temperature in Celsius to its equivalent in Fahrenheit. For the safety of your American team mate, round the result to the **next** `Integer`. Temperature in degrees Farenheit is defined as: T(°F) = T(°C) × 1.8 + 32. Your function should work like this:

Reviewer: ```suggestion
Write a second function, `tempToF` to convert a `Float` temperature in Celsius to its equivalent in Fahrenheit.
For safety reasons, round the result to the **next** `Integer`.
Temperature in degrees Farenheit is defined as: T(°F) = T(°C) × 1.8 + 32. Y.
```

# Instructions

You are part of a firefighter organization made of international volunteers.
Due to the international nature of the team, valuable time has been wasted over the last few calls translating between Fahrenheit and Celcius while estimating the temperature inside the burning buildings.

> To save time, and people, you decide to create a program to automatically convert between Fahrenheit and Celsius for your new teammate.

Reviewer: I don't think there is a teammate reference anymore

```suggestion
To save time, and people, you decide to create a program to automatically convert between Fahrenheit and Celsius for your international team.
```

## 1. Convert Farenheit to Celsius

> Write a function `tempToC` to convert an `Integer` temperature in Fahrenheit to its equivalent `Float` in Celsius rounded to the nearest hundredth. 

Reviewer: Tiny nit: the function stub is there, so it's more a "fill in the blanks" that writing the function.

```suggestion
Implement the `tempToC` function to convert an `Integer` temperature in Fahrenheit to its equivalent `Float` in Celsius rounded to the nearest hundredth. 
```

Temperature in degrees Celsius is defined as: T(°C) = (T(°F) - 32) / 1.8.

```haskell
> tempToC 32
0.00
```

## 2. Convert Celsius to Farenheit

> Write a second function, `tempToF` to convert a `Float` temperature in Celsius to its equivalent in Fahrenheit.

Reviewer: ```suggestion
Implement the `tempToF` function to convert a `Float` temperature in Celsius to its equivalent in Fahrenheit.
```
